consensus/polybft/fsm.go

```diff
@@ -69,12 +64,12 @@ type fsm struct {
 	// isEndOfSprint indicates if sprint reached its end
 	isEndOfSprint bool
 
+	// current epoch metadata
+	epoch *epochMetadata
```

Comment:
I would remove this field from `fsm`. I see you only use it when building bundles on `InsertBlock`. You can extract it from `runtime`.

Reply:
removed https://github.com/0xPolygon/polygon-edge/commit/f0d08eb1a44c7b7b182c3f1757dc2e855bee7d57